databases/week1/homework.sql

```diff
@@ -0,0 +1,39 @@
+/* 1. Find out how many tasks are in the task table */
+SELECT COUNT(*) AS task_count FROM task;
+
+/* 2. Find out how many tasks in the task table do not have a valid due date */
+SELECT COUNT(*) AS task_count_due_date_null FROM task WHERE due_date IS NULL;
+
+/* 3. Find all the tasks that are marked as done */
+SELECT * FROM task JOIN status ON task.status_id = status.id
+WHERE status.name = "Done";
+
+/* 4. Find all the tasks that are not marked as done */
+SELECT * FROM task JOIN status ON task.status_id = status.id
+WHERE status.name != "Done";
+
+/* 5. Get all the tasks, sorted with the most recently created first */
+SELECT * FROM task ORDER BY created DESC;
+
+/* 6. Get the single most recently created task */
+SELECT * FROM task ORDER BY created DESC LIMIT 1;
```

Comment:
Same as above :)

Reply:
I always prefer to get the full table, so I can see everything at the same time :D

Reply:
I cannot say you're wrong xD